bigquery/README.md

```diff
@@ -4,3 +4,55 @@ This notifier pushes build data to a BigQuery instance.
 
 *Alpha Feature - Under Development*
 
+## Accessing build insights with SQL queries through the BigQuery CLI:
+
```

Comment:
May seem obvious, but can we add one sentence here explain this line. Something like: "To access BiqQuery..., enter the following command into your terminal"

Reply:
@Aric1088 please add Navie's requested change.